src/Tests/AuthoringWinUITest/AuthoringWinUITest/Directory.Build.targets

```diff
@@ -1,16 +1,15 @@
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.targets', '$(MSBuildThisFileDirectory)../'))" />
+
   <PropertyGroup>
     <PrepareForRunDependsOn>CopyTestAssets;$(PrepareForRunDependsOn)</PrepareForRunDependsOn>
   </PropertyGroup>
 
-  <!-- temporary until nuget support -->
   <Target Name="CopyTestAssets">
-    <Copy SourceFiles="$(BuildOutDir)WinRT.Host\bin\WinRT.Host.dll"
-          DestinationFiles="$(OutDir)WinRT.Host.dll" 
+    <Copy SourceFiles="$(BuildOutDir)WinRT.Host\bin\WinRT.Host.dll.mui"
```

Comment:
DLL is automatically copied?

Reply:
Looks like we have to manually place in both the AuthoringWinUITest and in AuthoringWinUITest (Package) projects. Restored and added mui to (Package) wapproj